lib/components/Breadcrumb/Breadcrumb.tsx

```
@@ -0,0 +1,135 @@
+    ) => (
+      <StyledBreadcrumbItem key={key}>
+        <BreadcrumbItem 
+          {...item} 
+          isLast={isLast}
+        />
+        {!isLast && <StyledDivider>/</StyledDivider>}
+      </StyledBreadcrumbItem>
+    );
+
+    const renderFullBreadcrumb = () => (
+      <StyledBreadcrumbContainer ref={ref as any}>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)